tests/audio_flow_test.py

```
@@ -77,230 +74,162 @@ def on_console(msg) -> None:
+        )
 
-        # Check only new logs for no warnings, ignoring known encryption fallbacks
+        # Ensure no unexpected warning logs were generated
         new_logs = logs[pre_change_log_count:]
         for log in new_logs:
             text = log["text"].lower()
             if "warning" in text and "encryption aborted" not in text:
-                assert (
-                    False
-                ), f"Unexpected warning after music volume change: {log['text']}"
+                assert False, f"Unexpected warning after music volume change: {log['text']}"
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)